examples/conftest.yml

```diff
@@ -0,0 +1,91 @@
+# This example shows how to use compost with **Atlantis v0.18**, Conftest, and Infracost to enforce cost policies.
+
+# To run this:
+# 1. Add the yaml contents of this file to your `repos.yaml` or `atlantis.yaml` server side config file.
+# 2. On the Atlantis server, export env vars for:
+#    export GITHUB_TOKEN=<your-github-token>
+#    export INFRACOST_API_KEY=<your-infracost-api-token>
+# 3. Run the `infracost/infracost-atlantis` image, which includes the Infracost and Compost CLIs in addition to Atlantis:
+#    docker run -p 4141:4141 -e GITHUB_TOKEN=$GITHUB_TOKEN -e INFRACOST_API_KEY=$INFRACOST_API_KEY \
+#      --mount type=bind,source=$(pwd)/examples/monorepo_github_comment.yml,target=/home/atlantis/repo.yml \
+#      infracost/infracost-atlantis:latest \
+#      --gh-user=<your-github-user> \
+#      --gh-token=$GITHUB_TOKEN \
+#      --gh-webhook-secret=<your-github-webhook-secret> \
+#      --repo-allowlist='github.com/your-org/*' --repo-config=/home/atlantis/repo.yml
+# 4. Send a pull request in GitHub to change something in TF, the Infracost pull request comment should be added.
+# 5. Experiment with different cost policies by editing the policy file generated in the pre_workflow_hook.
+
+workflows:
+  terraform-infracost:
+    policy_check:
+      steps:
+        - env:
+            name: INFRACOST_JSON
+            command: 'echo $REPO_REL_DIR-$WORKSPACE-infracost.json'
+        - show
+        - run: echo "Generating Infracost cost estimates for $REPO_REL_DIR/$WORKSPACE..."
+        - run: infracost breakdown --path=$SHOWFILE --format=json --out-file=$INFRACOST_JSON --log-level=warn --no-color
```

Comment:
We can't use $PLANFILE here?

Reply:
Oooh nice, when I tried `$SHOWFILE` last it didn't work for me, but I think this is better than `$PLANFILE` since this is the plan JSON not the binary plan.

Reply:
Ah nice!

Reply:
I think the trick is you need to have the `-show` step before you can use $SHOWFILE